src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -220,7 +220,7 @@ protected void Initialize(ISchema schema, IServiceProvider serviceProvider, IEnu
             _introspectionTypes = CreateIntrospectionTypes(schema.Features.AppliedDirectives, schema.Features.RepeatableDirectives);
 
             _context = new TypeCollectionContext(
-               type => BuildGraphQLType(type, t => _builtInScalars.TryGetValue(t, out var graphType) ? graphType : _introspectionTypes.TryGetValue(t, out graphType) ? graphType : (IGraphType)Activator.CreateInstance(t)!),
+               type => BuildGraphQLType(type, ([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] t) => _builtInScalars.TryGetValue(t, out var graphType) ? graphType : _introspectionTypes.TryGetValue(t, out graphType) ? graphType : (IGraphType)Activator.CreateInstance(t)!),
```

Comment:
This is another situation where it might be more appropriate to mark the entire class as `[RequiresUnreferencedCode]`, and then ignore warnings from callers.

Reply:
> This is another situation where it might be more appropriate to mark the entire class as [RequiresUnreferencedCode], 

While I slightly understand the reasoning here I do not undestand the next part:

> and then ignore warnings from callers.